app/components/c-add-to-cart.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div :class="b()">
+    <e-input
+      ref="quantity"
+      :class="b('quantity')"
+      v-model="quantity"
+      name="quantity"
+      @input="onInput"
+    />
+    <e-button
+      :class="b('button')"
+      primary
+      width="auto"
+      @click="onClick"
+    ><e-icon icon="i-cart--blue"/></e-button>
+  </div>
+</template>
+
+<script>
+  import { mapActions } from 'vuex';
+
+  export default {
+    name: 'c-add-to-cart',
+    // components: {},
+    // mixins: [],
+
+    props: {
+      /**
+       * Sku of the article to be added
+       */
+      sku: {
+        type: String,
+        required: true,
+      },
+
+      /**
+       * To distinguish items with the same sku
+       */
+      groupKey: {
+        type: String,
+        default: '',
+      }
+    },
+    data() {
+      return {
+        quantity: '1',
```

Comment:
- [ ] Please use a Number value.
- [ ] I think we should also add a prop, because there could be a pre defined amount.